pgjdbc/src/main/java/org/postgresql/jdbc/ArrayDecoding.java

```diff
@@ -389,7 +447,7 @@ Object parseValue(String stringVal, BaseConnection connection) throws SQLExcepti
     OID_TO_DECODER.put(Oid.TIME, TIME_DECODER);
     OID_TO_DECODER.put(Oid.TIMETZ, TIME_DECODER);
     OID_TO_DECODER.put(Oid.TIMESTAMP, TIMESTAMP_DECODER);
-    OID_TO_DECODER.put(Oid.TIMESTAMPTZ, TIMESTAMP_DECODER);
+    OID_TO_DECODER.put(Oid.TIMESTAMPTZ, TIMESTAMPZ_DECODER);
```

Comment:
Don't change this in this PR. We will change it, but not here

Reply:
Should I go ahead and create another PR for that so I can rebase? Some tests depend on it.

Reply:
Not at the moment, let me look at the other PR's and see if they can be merged

Reply:
This does not need to change unless we are wanting to add support for binary representation of timestamp/timestampz.
This PR is misusing the ArrayDecoding api as a way to facilitate parsing singleton values out of structs.